benchmarks/README.md

```diff
@@ -0,0 +1,80 @@
+# Benchmarks for Optuna
```

Comment:
How about leaving a couple of words about what's ultimately generated, e.g. the plots and how to interpret them?
An example would be really nice.